l10n_ch_base_bank/models/bank.py

```diff
@@ -152,8 +152,8 @@ def name_search(self, name, args=None, operator='ilike', limit=80):
             ids = self.search(
                 args,
                 limit=limit
-                ).ids
```

Comment:
I find useless to search an empty value on all fields.
But it can be improved by constructing only the domain in the `if...else` and pass the domain to the same search.

``` python
if name:
    domain = [...] + args # using name
else:
    domain = args
ids = self._search(domain, limit=limit, ...)
```

We might even use `_search` to get directly the `ids` without using `browse` implied by `search`